Dell/recovery_gtk.py

```diff
@@ -136,11 +139,24 @@ def top_button_clicked(self, widget):
                     dbus_sync_call_signal_wrapper(self.backend(),
                                                   "enable_boot_to_restore",
                                                   {},
-                                                  False)
-                    proc = subprocess.Popen(["gnome-session-quit", "--reboot"])
+                                                  True)
+                    #proc = subprocess.Popen(["gnome-session-quit", "--reboot"])
+                    self.destroy()
+                except dbus.DBusException as msg:
+                    self.dbus_exception_handler(msg)
+                       
+            #Restore DHC Button
+            else:   
```

Comment:
So does that mean that this button will be present at all times on all machines?

Reply:
This button is only available on platforms which need recovery DHC and recovery normal Ubuntu.